code/game/gamemodes/game_mode.dm

```diff
@@ -361,6 +361,13 @@
 			heads += player.mind
 	return heads
 
+/datum/game_mode/proc/get_assigned_head_roles()
```

Comment:
I added this entirely for debug reasons but hey maybe someone will use it years from now